src/lib.rs

```diff
@@ -42,19 +49,20 @@ pub fn run(args: Args) {
             .capture_index_for_name(capture_name)
             .expect(&format!("Unknown capture name: `{}`", capture_name))
     });
+
     enumerate_project_files(&*supported_language)
         .par_iter()
-        .flat_map(|project_file_dir_entry| {
-            get_results(
-                &query,
-                project_file_dir_entry.path(),
-                capture_index,
-                language,
-            )
+        .for_each(|project_file_dir_entry| {
+            let mut printer = grep::printer::Standard::new_no_color(io::stdout());
```

Comment:
Prints out the matches, by default in a "normal" grep style, I guess, but is configurable via a builder to handle all the standard grep output options.